domain/src/main/java/org/oppia/android/domain/exploration/lightweightcheckpointing/ExplorationCheckpointController.kt

```diff
@@ -40,25 +41,14 @@ class ExplorationCheckpointController @Inject constructor(
   class ExplorationCheckpointNotFoundException(message: String) : Exception(message)
 
   /**
-   * These Statuses correspond to the exception and the checkpoint database states above
-   * such that if the deferred result contains
-   *
-   * CHECKPOINT_SAVED_DATABASE_SIZE_LIMIT_EXCEEDED, the
-   * [ExplorationCheckpointState.CHECKPOINT_SAVED_DATABASE_EXCEEDED_LIMIT] will be
-   * passed to a successful AsyncResult.
-   *
-   * CHECKPOINT_SAVED_DATABASE_SIZE_LIMIT_NOT_EXCEEDED,
-   * [ExplorationCheckpointState.CHECKPOINT_SAVED_DATABASE_NOT_EXCEEDED_LIMIT] will be
-   * passed to a successful AsyncResult.
+   * These Statuses correspond to the exception above such that if the deferred result contains
```

Comment:
This statement to me is a bit unclear. What `exception above` is this referring to? Maybe instead of mentioning it that way, provide the actual exception name

Reply:
changed the kdoc explanation.